namd/src/colvarproxy_namd.h

```diff
@@ -124,7 +124,7 @@ class colvarproxy_namd : public colvarproxy, public GlobalMaster {
   }
 
 #if CMK_SMP && USE_CKLOOP
-  int check_smp_enabled() override;
+  int check_smp_enabled(smp_mode_t mode) override;
```

Comment:
Since @jhenin previosly gave me grief about the confusion raised by the previous name of this function, I think it's only fair to point out that this is also a bit confusing. How about returning the `colvarproxy::smp_mode` via accessor?

Reply:
Bonus point if "off" evaluates to false, but I won't require it :-)

Reply:
I have moved `smp_mode` to the protected section of the class and provided getter and setter for it.